randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -524,14 +528,24 @@ Extra - asset_id: collision_camera_007
   > Door to Elun Transport Access
       Before Ghavoran - Super Missile Rotatable
   > Event - Super Missile Rotatable
-      Morph Ball and Before Ghavoran - Super Missile Rotatable
+      Morph Ball and Before Ghavoran - Super Missile Rotatable and Knowledge (Intermediate)
+  > Event - Super Missile Rotatable with Ledge Warp
+      All of the following:
+          Before Ghavoran - Super Missile Rotatable and Knowledge (Intermediate) and Ledge Warp (Intermediate) and Lay Bomb
+          After Ghavoran - Super Missile Area Access or Enabled Highly Dangerous Logic
```

Comment:
This doesn't need "Before Ghavoran - Super Missile Rotatable".  Just makes it dangerous when it doesnt need to be, for the same reason some "Before" events were removed in other prs, like the Dairon - Teleport to Artaria speed blocks.

Reply:
Makes, sense, I'll remove it from this one. However, the resource has to be dangerous because you can only rotate it the intended way once, and there's a connection below the spinner that is only available before the rotation. The connections that are relevant when entering through the door at the bottom before rotating the spinner could have misc resource on them, because they are only relevant in transport rando.